README.md

```diff
@@ -79,24 +79,24 @@ Thats it!
 Learn more about Ketch at [Ketch documentation](https://learn.theketch.io/docs)
 
 ### Quick Start
-Deploying apps is easy once you've installed Ketch.  First, create a pool. Then create app(s) adding them to the pool and finally
+Deploying apps is easy once you've installed Ketch.  First, create a framework. Then create app(s) adding them to the framework and finally
 deploy the app(s).  The following example illustrates these steps.
 
 ```bash
-# Add a pool with ingress Traefik (default), replace ingress IP address by your ingress IP address
-ketch pool add mypool  --ingress-service-endpoint 35.247.8.23 --ingress-type traefik
+# Add a framework with ingress Traefik (default), replace ingress IP address by your ingress IP address
+ketch framework add myframework  --ingress-service-endpoint 35.247.8.23 --ingress-type traefik
 
 # Create app
-ketch app create bulletinboard --pool mypool
+ketch app create bulletinboard --framework myframework
```

Comment:
The `create` command is gone in 0.3, rather it's functionality merged into `deploy`, below. So this can be removed.